ffd_info_exchange/fafsa/static/assets/css/main.css

```diff
@@ -118,10 +135,10 @@ label {
     color: #ffffff;
     display: block;
     font-size: 14px;
-    left: 50%;
+    left: 20px;
```

Comment:
I believe the percentage based positioning is using the size of the tooltip icon as it's positioning so to get it to the appropriate place it would have needed to be 200% or something which just felt weird. I opted for the pixels since we know the constraints of our test machine on monday. 

For the second thing... good question. I was just following the lead for all the other styles that were being used by this prototype but maybe they should be moved to a more neutral place (`ffd_info_exchange`?) since both prototypes share styles.